docker-compose/docker-compose.yaml

```diff
@@ -252,29 +246,75 @@ services:
       - sourcegraph
     restart: always
 
-  # Description: LSIF HTTP server for code intelligence.
+  # Description: Serves precise code intelligence requests.
   #
-  # Disk: 200GB / persistent SSD
-  # Ports exposed to other Sourcegraph services: 3186/TCP (server) 3187/TCP (worker)
+  # Ports exposed to other Sourcegraph services: 3186/TCP (server)
   # Ports exposed to the public internet: none
   #
-  lsif-server:
-    container_name: lsif-server
-    image: 'index.docker.io/sourcegraph/lsif-server:3.14.0-1@sha256:ed683d2579d807458a892eab2f88fa5618b82a1ae7af02d5e7de1af3f6eea4a7'
+  precise-code-intel-api-server:
+    container_name: precise-code-intel-api-server
+    image: 'index.docker.io/sourcegraph/precise-code-intel-api-server:insiders'
     cpus: 2
     mem_limit: '2g'
     environment:
-      - GOMAXPROCS=2
-      - LSIF_STORAGE_ROOT=/lsif-storage
+      - PRECISE_CODE_INTEL_BUNDLE_MANAGER_URL=http://precise-code-intel-bundle-manager:3187
       - 'SRC_FRONTEND_INTERNAL=sourcegraph-frontend-internal:3090'
     healthcheck:
       test: "wget -q 'http://127.0.0.1:3186/ping' -O /dev/null || exit 1"
       interval: 5s
       timeout: 5s
       retries: 1
       start_period: 60s
+    networks:
+      - sourcegraph
+    restart: always
+
+  # Description: Stores and manages precise code intelligence bundles.
+  #
+  # Disk: 200GB / persistent SSD
+  # Ports exposed to other Sourcegraph services: 3187/TCP (manager)
+  # Ports exposed to the public internet: none
+  #
+  precise-code-intel-bundle-manager:
+    container_name: precise-code-intel-bundle-manager
+    image: 'index.docker.io/sourcegraph/precise-code-intel-bundle-manager:insiders'
```

Comment:
It's OK to merge this with `insiders` tags, but they need to be pinned using SHA digests.

Reply:
Updated to same commit as the others.